pandas/core/frame.py

```diff
@@ -3161,6 +3167,21 @@ def query(self, expr, inplace=False, **kwargs):
         For further details and examples see the ``query`` documentation in
         :ref:`indexing <indexing.query>`.
 
+        *Backtick quoted variables*
+
+        Backtick quoted variables are parsed as literal Python code and
```

Comment:
can you add a reference here to the python identifier docs

Reply:
OK